src/frontend/src/components/DetailDisplay.tsx

```diff
@@ -4,22 +4,27 @@
  */
 
 import Typography from '@mui/material/Typography';
+import { Box } from '@mui/system';
 
 interface DetailDisplayProps {
   label: string;
   content: string;
   paddingRight?: number;
+  copyButton?: React.ReactElement;
 }
 
-const DetailDisplay: React.FC<DetailDisplayProps> = ({ label, content, paddingRight = 0 }) => {
+const DetailDisplay: React.FC<DetailDisplayProps> = ({ label, content, paddingRight = 0, copyButton }) => {
```

Comment:
you can do copyButton = false in this line, and still allow it to be undefined, this'll cause it to default to false unless you specify otherwise